tests/composer/composer-symfony6-max.json

```diff
@@ -29,7 +29,7 @@
     "require-dev": {
         "ext-pdo": "*",
         "ext-json": "*",
-        "monolog/monolog": "^1.3",
+        "monolog/monolog": "^2.3",
```

Comment:
Is that an outdated branch?
https://github.com/propelorm/Propel2/blob/a2a4a39155bb1e27548a3e38f4f687f59a14257e/composer.json#L32
We should be able to keep both majors if we add the type

Reply:
It's a tests related composer.json. Those are quite outdated for dependencies which doesn't come from Symfony.

Probably those composer.json files should be replaced by something which use `symfony/flex` (for the `extra.symfony.require` flag and some `composer up --prefer-lower`).